array-manipulation-2/src/chunk.js

```diff
@@ -0,0 +1,10 @@
+/* exported chunk */
+
+function chunk(array, size) {
+  var Array = [];
```

Comment:
Make sure that your variable names are lowercase. You are actually overwriting the `Array` class here!

Reply:
Got it & corrected it accordingly. Thanks.